src/media/css/account.styl

```diff
@@ -81,6 +81,10 @@
         display: none;
     }
 }
+.html-rtl .region span {
```

Comment:
There's also an added level of modularity when using those block mixins, whereas you can redefine in the future what "RTL" means at an implementation level by changing a single mixin, rather than grepping across the project for every instance of that class. That's not so relevant here, but much more so if creating e.g. `mq--tablet()`.